src/Components/App/App.js

```diff
@@ -35,22 +38,32 @@ class App extends Component {
     return (
       <>
         <Header />
+      
         {this.state.error !== "" && <h2 className='error-message'>{this.state.error}</h2>}
 
-        <Route exact path='/' 
-          render={ () => { 
-            return <Movies 
+        <Switch >
```

Comment:
Not sure if the Switch was the only way to do this, but smart and works well to render the footer links the way we wanted them!